builtin/logical/transit/backend_test.go

```diff
@@ -587,15 +602,18 @@ func TestPolicyFuzzing(t *testing.T) {
 		}
 		fd := &framework.FieldData{}
 
+		var chosenFunc, chosenKey string
+
+		//t.Errorf("Starting %d", id)
 		for {
 			// Stop after 10 seconds
 			if time.Now().Sub(startTime) > 10*time.Second {
 				return
 			}
 
 			// Pick a function and a key
-			chosenFunc := funcs[rand.Int()%len(funcs)]
-			chosenKey := keys[rand.Int()%len(keys)]
+			chosenFunc = funcs[rand.Int()%len(funcs)]
```

Comment:
Generally yes, although normally I don't sweat minor optimizations in testing code :-D